views/trust-historical-beneficial-owner.html

```diff
@@ -0,0 +1,70 @@
+{% extends "layout.html" %}
+
+{% set title = "Historical beneficial owner" %}
+
+{% block pageTitle %}
+  {% include "includes/page-title.html" %}
+{% endblock %}
+
+{% block backLink %}
+  {% include "includes/back-link.html" %}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      {% include "includes/list/errors.html" %}
+
+      <span class="govuk-caption-l">{{ pageData.trustName }}</span>
+      <h1 class="govuk-heading-xl">{{ pageParams.title }}</h1>
+
+      <form class="form" method="post">
+        {% set fieldParam = {
+          name: 'name',
+          hint: 'You can add more later.',
+          label: 'What is the beneficial owner’s name?',
+          value: pageData.name,
+          error: errors.name
+        } %}
+        {% include "includes/inputs/text-input.html" %}
+
+        {% set fieldParam = {
+          name: 'createdDate',
+          label: 'When did they become a beneficial owner?',
+          hint: 'For example, 27 3 2007',
+          error: errors.createdDate,
+          value: {
+            'Day': pageData.createdDateDay,
+            'Month': pageData.createdDateMonth,
+            'Year': pageData.createdDateYear
+          }
+        } %}
+        {% include "includes/inputs/date-input.html" %}
+
+        {% set fieldParam = {
+          name: 'createdDate',
```

Comment:
notifiedDate

Reply:
We are planning on using field names that are consistent with the what is being asked for on the web page. 
I have updated these to "startDate" and "endDate". 
We'll map these to the relevant fields next story